### DISCUSSION TITLE
Welcome to dep-scan Discussions!

### DISCUSSION THREAD

**Discussion by prabhu** [owasp-dep-scan/dep-scan#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asjrosabal** [comment]
Hello, I'm trying to scan a java multiproject. But I doesn't obtain a result. I'm using a simple command: depscan --src $PWD --report_file $PWD/reports/depscan.json. How can I scan this kind of project? I have three build.gradle files, one of them in root of folder, and the others inside specific project folder.

![image](https://user-images.githubusercontent.com/15805802/147117102-615d8e9e-1c7c-4289-9332-cf1cbb618de7.png)
![image](https://user-images.githubusercontent.com/15805802/147117139-677a68b1-441e-46cd-98ae-ee312faefe6f.png)

**Reply by asjrosabal** [reply]
Hi @prabhu, Can you help me? Thanks

**Reply by prabhu** [reply]
@asjrosabal Have you installed cdxgen prior to invoking depscan?

```
sudo npm install -g @appthreat/cdxgen
```
